tests/test_parmparse.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+import os
+
+import amrex.space3d as amr
+
+
+def test_parmparse():
+    dopml = False
+    ncell = 10
+    dt = 0.0
+    pp = amr.ParmParse("")
+    dir_name = os.path.dirname(__file__)
+    pp.addfile(dir_name + "/parmparse_inputs")
+    pp_param = amr.ParmParse("param")
+    (_, ncell) = pp_param.query_int("ncell")
```

Comment:
This should work for unpacking tuples in Python:
```suggestion
    _, ncell = pp_param.query_int("ncell")
```